services/grpc.go

```diff
@@ -10,10 +10,26 @@ import (
 	"github.com/kacperf531/sockchat/api"
 	pb "github.com/kacperf531/sockchat/protobuf"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
 	emptypb "google.golang.org/protobuf/types/known/emptypb"
 )
 
+var GRPCCodes = map[error]codes.Code{
```

Comment:
Please also handle edge case with error not found in the map - zero value for `codes.Code` is `OK Code = 0`.

Error not found in the map is an unexpected internal error that should be logged for developers